build.sh

```diff
@@ -22,6 +22,7 @@ FORCE=0          # Force clean build flag
 VERBOSE=0        # Verbose output flag
 QUICK=${QUICK:-0} # Quick test mode (subset of tests)
 COV=${COV:-0}    # Coverage mode (for building and testing)
+SVS_SHARED_LIB=${SVS_SHARED_LIB:-0} # Build SVS as a shared library
```

Comment:
we need to rename this flag in VecSim 
```suggestion
SVS_SHARED_LIB=${SVS_SHARED_LIB:-0} # Use pre-compiled SVS binary
```